### ISSUE TITLE
Plugin hides inline title

### ISSUE THREAD

**Issue by RenanSantos7** [noatpad/obsidian-banners#101]
Since Obsidian 1.0 there is the inline tilte feature and I'd rather use it that way.

But, Banners hide it. Please fix this problem. Thanks!

**Comment by CobaltKyro** [comment]
Yep... it's been a few months, still present. Anyone able to update this plugin???

**Comment by robertandrews** [comment]
Any prospect of a fix?

**Comment by Thomas-Schmall** [comment]
I think this is a duplicate of https://github.com/noatpad/obsidian-banners/issues/91

There are also temp solutions noted. Although it would be nice to have this permanently fixed.
